### PR TITLE
ADOPTERS: elaborate on ovirt/okd integration

### PR DESCRIPTION
**Pull request by dankenigsberg**
Make it a bit clearer how oVirt and okd integrate with KubeVirt

/cc @sandrobonazzola @aburdenthehand 

```release-note
NONE
```

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
/lgtm

File: ADOPTERS.md [hunk 1]
```diff
@@ -8,8 +8,8 @@
 | End-user | Killercoda | 2022 | [link](https://killercoda.com) | Killercoda provides interactive learning environments based on VMs managed by KubeVirt. |
 | End-user | NVIDIA | 2018 | [link](https://www.nvidia.com) | NVIDIA's latest computing platform is built on open-source projects like Kubernetes and KubeVirt to power products like [GeForce NOW](https://www.nvidia.com/en-us/geforce-now/) with more to come. |
 | Integration | minikube | 2020 | [link](https://minikube.sigs.k8s.io) | |
-| Integration | oVirt | | [link](https://www.ovirt.org/documentation/administration_guide/index.html#proc-adding-kubevirt-openshift-as-an-external-provider_external_providers) | |
-| Integration | okd | 2020 | [link](https://www.okd.io) | |
+| Integration | oVirt | 2020 | [link](https://www.ovirt.org/documentation/administration_guide/index.html#proc-adding-kubevirt-openshift-as-an-external-provider_external_providers) | oVirt can view and manage VMs that run on a KubeVirt cluster|
```
Comment (aburdenthehand) [inline ADOPTERS.md:11]: oVirt can view and manage VMs that are running on a KubeVirt cluster
```suggestion
| Integration | oVirt | 2020 | [link](https://www.ovirt.org/documentation/administration_guide/index.html#proc-adding-kubevirt-openshift-as-an-external-provider_external_providers) | oVirt can view and manage VMs that run on a KubeVirt cluster|
```
```suggestion
| Integration | oVirt | 2020 | [link](https://www.ovirt.org/documentation/administration_guide/index.html#proc-adding-kubevirt-openshift-as-an-external-provider_external_providers) | oVirt can view and manage VMs that are running on a KubeVirt cluster. |
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
+| Integration | okd | 2020 | [link](https://www.okd.io) | [OKD Virtualization](https://docs.okd.io/latest/virt/about-virt.html) adds KubeVirt functionality to OKD |
```
Comment (aburdenthehand) [inline ADOPTERS.md:12]:
```suggestion
| Integration | okd | 2020 | [link](https://www.okd.io) | [OKD Virtualization](https://docs.okd.io/latest/virt/about-virt.html) adds KubeVirt functionality to OKD. |
```

**Review by aburdenthehand — COMMENTED** [review]
Thanks @dankenigsberg - appreciated!
I've suggested a slight reword on one, and added a fullstop on both for consistency.

**Review by sandrobonazzola — APPROVED** [review]
/lgtm

**Comment by aburdenthehand** [comment]
@dankenigsberg can you please sign off on your commit.
Otherwise the content lgtm. Thanks again

**Review by aburdenthehand — APPROVED** [review]

**Comment by aburdenthehand** [comment]
Unassigning because I don't have permissions to approve 
@stu-gott PTAL

**Pull request merged**